pyspark_ai/tool.py

```diff
@@ -104,3 +104,104 @@ def _run(
 
     async def _arun(self, *args: Any, **kwargs: Any) -> Any:
         raise NotImplementedError("ListTablesSqlDbTool does not support async")
+
+
+class ColumnQueryTool(BaseTool):
+    """Tool for finding the correct column name given keywords from a question."""
+
+    spark: Union[SparkSession, ConnectSparkSession] = Field(exclude=True)
+    name = "get_column_name"
+    description = """
+    This tool determines which column contains a keyword from the question.
+    Input to this tool is a str with a keyword of interest and a temp view name, output is the column name from the df
+    that contains the keyword.
+    Input should be pipe-separated, in the format: keyword|temp_view_name
```

Comment:
Shall we specify `args_schema` and use two args in the _run method?
Example: https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/tools/file_management/copy.py#L25